traefik/templates/_podtemplate.tpl

```diff
@@ -53,6 +53,9 @@
       - image: {{ template "traefik.image-name" . }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         name: {{ template "traefik.fullname" . }}
+        {{- if .Values.deployment.runtimeClassName }}
+        runtimeClassName: {{ .Values.deployment.runtimeClassName }}
+        {{- end }}        
```

Comment:
```suggestion
        {{- with .Values.deployment.runtimeClassName }}
        runtimeClassName: {{ . }}
        {{- end }}        
```

Reply:
@eddykaya do you want to add those suggestions before we merge your PR ? 
For consistency purpose only I would add @mloiseleur's suggestions but that's up to you :-)